README.md

```diff
@@ -14,20 +14,27 @@ documented in the `.msg` or `.srv` file associated with that interface.
 ROS messages and services used across many ROS packages in the project.
 
 ### Project-wide External Interfaces
-
+<!---
+Formatting reminder:
+1. Keep 'Type' column organized alphabetically
+2. 'mock' version of topic should be after its common counterpart
+--->
 | Topic                  | Type           | Publisher                | Subscriber(s)                               |
 | ---------------------- | -------------- | ------------------------ | ------------------------------------------- |
-| `ais_ships`            | AISShips       | AIS Receiver             | Local Pathfinding                           |
-| `batteries`            | Batteries      | CAN Transceiver          | Local Transceiver                           |
+| `ais_ships`            | AISShips       | CANTransceiverROSIntf    | Local Pathfinding, Local Transceiver        |
+| `mock_ais_ships`       | AISShips       | Mock AIS                 | CanSimIntf                                  |
+| `batteries`            | Batteries      | CANTransceiverROSIntf    | Local Transceiver                           |
+| `boat_cmd`             | BoatCmd        | Controller               | CANTransceiverROSIntf                       |
 | `desired_heading`      | DesiredHeading | Local Pathfinding        | Controller, Boat Simulator                  |
-| `data_sensors`         | GenericSensors | CAN Transceiver          | Local Transceiver                           |
-| `global_path`          | Path           | Local Transceiver        | Local Pathfinding                           |
-| `gps`                  | GPS            | CAN Transceiver          | Local Transceiver, Local Pathfinding        |
+| `data_sensors`         | GenericSensors | CANTransceiverROSIntf    | Local Transceiver                           |
+| `gps`                  | GPS            | CANTransceiverROSIntf    | Local Transceiver, Local Pathfinding        |
+| `mock_gps`             | GPS            | Boat Simulator           | CanSimIntf                                  |
 | `local_path_data`      | LPathData      | Local Pathfinding        | Local Transceiver                           |
-| `mock_gps`             | GPS            | Boat Simulator           | CAN Transceiver                             |
-| `filtered_wind_sensor` | WindSensor     | CAN Transceiver          | Local Transceiver, Local Pathfinding        |
-| `mock_wind_sensors`    | WindSensors    | Boat Simulator           | CAN Transceiver                             |
-| `wind_sensors`         | WindSensors    | CAN Transceiver          | Local Transceiver                           |
+| `global_path`          | Path           | Local/Remote Transceiver | Local Pathfinding                           |
+| `sim_cmd`              | SimCmd         | CanSimIntf               | Simulator                                   |
+| `filtered_wind_sensor` | WindSensor     | CANTransceiverROSIntf    | Local Transceiver, Local Pathfinding        |
```

Comment:
Let's use the abbreviation for transceiver and use camelcase: `CanTrxRosIntf`

Reply:
Replaced CANTransceiverROSIntf with CanTrxRosIntf